src/oneD/Boundary1D.cpp

```diff
@@ -707,9 +705,17 @@ void ReactingSurf1D::eval(size_t jg, double* xg, double* rg,
         double* xb = x - nc;
         rb[c_offset_T] = xb[c_offset_T] - m_temp; // specified T
         size_t nSkip = m_flow_left->rightExcessSpecies();
+        size_t l_offset = 0;
+        thermo_t* left_thermo = &m_flow_left->phase();
```

Comment:
```suggestion
        ThermoPhase* left_thermo = &m_flow_left->phase();
```
I would eventually like to eliminate these typedefs for the core Cantera types -- they just introduce unnecessary confusion.